test/integration/targets/lookup_mongodb/tasks/test_lookup_mongo.yml

```diff
@@ -0,0 +1,62 @@
+- name: "assert we complain properly if pymongo is not there"
+  debug: msg="{{ lookup('mongodb', mongodb_parameters ) }}"
+  register: result
+  ignore_errors: yes
+
+- name: "assert we complain properly if pymongo is not there (assert part)"
+  assert:
+    that:
+    - result is failed
+    - result is not changed
+    - "'pymongo is required' in result.msg"
+
+- name: install pymongo
+  pip:
+    name: "pymongo"
+
+- name: "assert we complain properly if we can't connect to mongo"
+  debug: msg="{{ lookup('mongodb', mongodb_parameters ) }}"
+  register: result
+  ignore_errors: yes
```

Comment:
Can the connection timeout be lowered? This task takes 30 seconds to complete:

```
13:42 TASK [lookup_mongodb : assert we complain properly if we can't connect to mongo] ***
14:12 There was an exception on the mongodb_lookup: unable to connect to database: localhost:27017: [Errno 111] Connection refused
```

Reply:
Agreed. I changed it to 5 seconds in the test.